python/ray/experimental/state/common.py

```diff
@@ -506,6 +530,15 @@ class ClusterEventState(StateSchema):
 class TaskState(StateSchema):
     """Task State"""
 
+    @classmethod
+    def humanify(cls, state: dict) -> dict:
+        if not cls.is_valid_state(state):
```

Comment:
Wondering if we could move this logic to the parent state, the Abstract state class.

Reply:
We could! I originally was thinking each subclass would have to define its own humanify logic. But after the design has been flushed out a bit more, there isn't a need to re-define in the subclasses. I can push an update!